java/configuration/src/main/java/sleeper/configuration/properties/instance/CommonProperty.java

```diff
@@ -194,6 +194,19 @@ public interface CommonProperty {
             .validationPredicate(Utils::isTrueOrFalse)
             .propertyGroup(InstancePropertyGroup.COMMON)
             .runCdkDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty TRANSACTION_LOG_SNAPSHOT_CREATION_BATCH_SIZE = Index.propertyBuilder("sleeper.metadata.transactionlog.snapshot.creation.batch.size")
+            .description("The number of tables to create transaction log snapshots for in a single invocation. A separate invocation " +
+                    "of the lambda will be made for each batch when there are more tables than the batch size.")
+            .defaultValue("5")
+            .validationPredicate(Utils::isPositiveInteger)
+            .propertyGroup(InstancePropertyGroup.COMMON)
+            .build();
+    UserDefinedInstanceProperty TRANSACTION_LOG_SNAPSHOT_CREATION_LAMBDA_PERIOD_IN_MINUTES = Index.propertyBuilder("sleeper.metadata.transactionlog.snapshot.creation.lambda.period.minutes")
+            .description("The frequency in minutes with which the transaction log snapshot creation lambda is run.")
+            .defaultValue("5")
```

Comment:
I'm not sure if it might be worth making this seconds rather than minutes, just in case?

Reply:
The other lambda invocation period properties use minutes. I think its best to stay consistent

Reply:
The property name says seconds now, it should be consistent.

I think the only question is whether someone might want to fine tune it to that degree. We can always change it later.